vars/openshiftEnsureGenericBinaryBuild.groovy

```diff
@@ -0,0 +1,25 @@
+def call () {
+  def genericBuildTemplate = """{
+    "apiVersion": "v1",
+    "kind": "BuildConfig",
+    "metadata": {
+        "name": "generic-component-build"
+    },
+    "spec": {
+        "output": {},
+        "runPolicy": "Serial",
```

Comment:
If all builds are sharing this one `generic-component-build` BuildConfig, and the runPolicy is serial here; what will happen when two or more component jobs trigger a build on this? Will those builds queue up, or will running ones be cancelled in favour of newer ones? Are any of these cases actually desirable?